components/Radio/RadioGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {
+  SFC,
+  ReactNode,
+  CSSProperties,
+  useState,
+  cloneElement,
+} from "react"
+import _ from "lodash"
+import { RadioProps } from "./Radio"
+
+export interface RadioChangeEventTarget extends RadioProps {
+  checked: boolean
+}
+
+type size = "small" | "medium"
+
+export interface RadioChangeEvent {
+  target: RadioChangeEventTarget
+  stopPropagation: () => void
+  preventDefault: () => void
+  nativeEvent: MouseEvent
+}
+
+interface RadioGroupProps {
+  /** Value of the radio group */
+  value?: any
+  /** Custom styles for each radio button */
+  radioStyle?: CSSProperties
+  /** Function to call when radio is changed */
+  onChange?: (e: RadioChangeEvent) => void
+  /** default value of the group */
+  defaultValue?: any
```

Comment:
`defaultValue` should not be there, it should be determined based on which option gets selected by default.

Reply:
This can be important when we want to show one option is selected by default, do you have a better way to implement this? 

```
<RadioGroup defaultValue={1}>
  <Radio.Item value={2} />
  <Radio.Item value={1} />
</RadioGroup>
```

Reply:
How about having a `default` prop on the children? In the case of the multiple chilrend with default, just choose the last one which has a default. I like this approach slightly better because it is similar to the way it is done in native HTML. If we can use the `checked` prop for default selection it would even better.

Reply:
lgtm, nice

Did the 2nd way suggested, getting native feels now ❄️